### ISSUE TITLE
Add `mix hex.add PACKAGE` task

### ISSUE THREAD

**Issue by webdeb** [hexpm/hex#694]

**Comment by supersimple** [comment]
hi @webdeb 👋 
can you describe what this task should do? how it differs from hex.publish for instance?

thanks

**Comment by webdeb** [comment]
hi @supersimple,
the idea is to install dependencies directly from the command line.

```
mix hex.add jason
# OR
mix deps.add jason
```

I know this would require a `/deps.exs` file. But would't it make sense?

**Comment by supersimple** [comment]
IMO using mix implies you have a mix.exs file with dependencies in it, so any task that would do this would have to modify that file (adding a dependency). In my mind it would:
1. add a line to dependencies in the mix file
2. run mix deps.get

Either way, I _do_ think this issue or discussion belongs in the Elixir project https://github.com/elixir-lang/elixir/issues

**Comment by wojtekmach** [comment]
Hi @webdeb, this has been proposed a few times in the past, here's one example discussion about such feature: https://elixirforum.com/t/mix-deps-add-functionality/4856

TL;DR: it's very tricky because it requires rewriting `mix.exs`. For this reason, I think folks should try to solve these problems in a 3rd-party tool and then potentially propose it to Hex/Elixir. Thanks!
